Clean-up/Terraform/main.tf

```diff
@@ -0,0 +1,79 @@
+# This file terraforms lambda and eventbridge on AWS
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.AWS_REGION
+}
+
+# terraform import aws_lambda_function.c11-hermes-clean_lambda c11-hermes-clean_lambda
+resource "aws_lambda_function" "c11-hermes-clean_lambda" {
+    function_name         = "c11-hermes-clean_lambda"
+    image_uri             = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c11-hermes-clean@sha256:e9ccc95322d51a339b97f61fc1f109b1f9f0d21a4598c3d470c6f857328587ca"
+    package_type          = "Image"
+    architectures         = ["x86_64"]
+    role                  = aws_iam_role.lambda_role.arn
+
+    environment {
+        variables = {
+            "DB_HOST"     = var.DB_HOST
+            "DB_NAME"     = var.DB_NAME
+            "DB_PASSWORD" = var.DB_PASSWORD
+            "DB_PORT"     = var.DB_PORT
+            "DB_USER"     = var.DB_USER
+        }
+    }
+
+    logging_config {
```

Comment:
I think logging is done automatically so this is not needed
https://docs.aws.amazon.com/lambda/latest/dg/python-logging.html

Reply:
I guess, it's good to have